openbb_terminal/mutual_funds/mutual_fund_controller.py

```diff
@@ -225,330 +581,192 @@ def call_search(self, other_args: List[str]):
-            nargs="+",
             required="-h" not in other_args,
         )
-        parser.add_argument(
-            "-n",
-            "--name",
-            action="store_true",
-            default=False,
-            dest="name",
-            help="Flag to indicate name provided instead of symbol.",
-        )
-        # Keeping the date format constant for investpy even though it needs to be reformatted in model
         parser.add_argument(
             "-s",
             "--start",
             type=valid_date,
-            default=(datetime.now() - timedelta(days=366)).strftime("%Y-%m-%d"),
+            default=(datetime.now() - timedelta(days=1100)).strftime("%Y-%m-%d"),
             dest="start",
-            help="The starting date (format YYYY-MM-DD) of the fund",
+            help="The starting date (format YYYY-MM-DD) of the stock",
         )
         parser.add_argument(
             "-e",
             "--end",
             type=valid_date,
             default=datetime.now().strftime("%Y-%m-%d"),
             dest="end",
-            help="The ending date (format YYYY-MM-DD) of the fund",
+            help="The ending date (format YYYY-MM-DD) of the stock",
         )
         if other_args and "-" not in other_args[0][0]:
             other_args.insert(0, "--fund")
         ns_parser = self.parse_known_args_and_warn(parser, other_args)
         if ns_parser:
-            parsed_fund = " ".join(ns_parser.fund)
-            (
-                self.data,
-                self.fund_name,
-                self.fund_symbol,
-                self.country,
-            ) = investpy_model.get_fund_historical(
-                name=parsed_fund,
-                by_name=ns_parser.name,
-                country=self.country,
-                start_date=ns_parser.start,
-                end_date=ns_parser.end,
+            funds_loaded = mstarpy_view.display_load(
+                term=ns_parser.fund, country=self.country
             )
-            if self.data.empty:
-                console.print(
-                    """No data found.
-Potential errors
-    -- Incorrect country specified
-    -- ISIN supplied instead of symbol
-    -- Name used, but --name flag not passed"""
-                )
+            self.funds_loaded = funds_loaded
+            self.fund_name = funds_loaded.name
+            self.fund_symbol = funds_loaded.code
+            self.end_date = ns_parser.end.strftime("%Y-%m-%d")
+            self.start_date = ns_parser.start.strftime("%Y-%m-%d")
         return self.queue
 
     @log_start_end(log=logger)
     def call_plot(self, other_args: List[str]):
-        """Process country command"""
+        """Process plot command"""
         parser = argparse.ArgumentParser(
             add_help=False,
             formatter_class=argparse.ArgumentDefaultsHelpFormatter,
             prog="plot",
             description="Plot historical data.",
         )
 
+        parser.add_argument(
+            "-comp",
```

Comment:
By convention, we just use one letter with a single dash, so `-c` should be used here